client/components/plans/plans-compare/index.jsx

```diff
@@ -200,13 +204,13 @@ const PlansCompare = React.createClass( {
 
 		// add google-ad-credits feature
 		if ( this.isUSorCanada() && isGoogleVouchersEnabled() ) {
-			features.splice( 1, 0, googleAdCreditsFeature );
+			features.splice( 1, 0, googleAdVouchersFeature );
 		}
 		// update the description if we are also including wordpressAdCredits
 		// this is necessary here so we don't call the abtest module outside of render
 		if ( this.isUSorCanada() && isWordpressAdCreditsEnabled() ) {
-			googleAdCreditsFeature.compareDescription = googleAdCredits.getDescriptionWithWordAdsCredit();
-			googleAdCreditsFeature[ '1008' ] = '$200';
+			googleAdVouchersFeature.compareDescription = googleWordAdsAdVouchers.getDescription();
+			googleAdVouchersFeature[ '1008' ] = '$200';
```

Comment:
Reading the comments this becomes more clear. It is a bit confusing to see this in the code though. For example we could instead have a voucher feature with an array of voucher sources and amount values, which we sum for the total credit.